python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -145,7 +149,7 @@ def _common_submit(self, dirac_script):
         self.statusInfo = ''
         j.been_queued = False
         dirac_cmd = """execfile(\'%s\')""" % dirac_script
-        result = execute(dirac_cmd)
+        result = execute(dirac_cmd, cred_req=self.credential_requirements)
```

Comment:
Atm I'm unable to see where the credential requirement is defined, probably because of something fancy in the decorator.
I'm reading the code passing an undeclared parameter rather than some new execute function where the credentials are worked out.